wallet-new/src/Cardano/Wallet/Kernel/Submission/Worker.hs

```diff
@@ -0,0 +1,41 @@
+module Cardano.Wallet.Kernel.Submission.Worker (
+    tickSubmissionLayer
+    ) where
+
+import           Universum
+
+import           Control.Concurrent (threadDelay)
+import qualified Data.Set as Set
+import           Formatting (sformat, (%))
+import qualified Formatting as F
+
+import           Cardano.Wallet.Kernel.Submission (Evicted, WalletSubmission)
+
+import           System.Wlog (Severity (..))
+
+
+tickSubmissionLayer :: forall m. (MonadCatch m, MonadIO m)
+                    => (Severity -> Text -> m ())
```

Comment:
If it is useful to have additional context for the log messages, feel free to change this callback and give it an additional context argument (you'll need to propagate that through).